action.yml

```diff
@@ -53,6 +53,19 @@ inputs:
     description: 'When set to true, disables verification of the Vault server certificate.  Setting this to true in production is not recommended.'
     required: false
     default: "false"
+  role:
```

Comment:
Thinking these should all be prefixed with the name of the auth method as we look to add more Auth Methods.

For example:
`jwtRole`
`jwtPrivateKey`
`jwtKeyPassword`
`jwtTTL`

Thoughts?

Reply:
Multiple auth methods will have `role` argument, but i can't imagine scenario, where multiple roles will be used simultaneously. So, i thought that arguments can be reusable between auth methods, if needed.

But your suggestion also valid, so i will accept you decision on this question

Reply:
I would almost say leave role as is and remove `JWT` from it's description. For example, the #155 PR will also feature role and there's no meaningful difference between role params for the different engines. Agree on other params though that aren't generic.

Reply:
The final version is:
`role`
`jwtPrivateKey`
`jwtKeyPassword`
`jwtTtl`